### DISCUSSION TITLE
Welcome to pomotroid Discussions!

### DISCUSSION THREAD

**Discussion by Splode** [Splode/pomotroid#142] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by naks110** [comment]
What if we make the whole window appear from taskbar notification icon itself? Its more easier to manage, especially for people like me who have lots of windows open.

If Pomotroid comes itself as a window, then my workflow might get distracted.
Presently it has a option of closing & minimizing to taskbar notification area.

I am attaching a screenshot of the UI that pops up when I hover my mouse over one of my fave programs (Auslogics BoostSpeed). Its not just a simple text menu, but an entire UI consisting of graphics, graphs & buttons - all on the notification area.

![Screenshot (369)](https://user-images.githubusercontent.com/33699454/131214144-d8bdec60-0ee9-468c-a399-03ce908e1745.png)

**Reply by y-martini** [reply]
> What if we make the whole window appear from taskbar notification icon itself? Its more easier to manage, especially for people like me who have lots of windows open.
> 
> If Pomotroid comes itself as a window, then my workflow might get distracted. Presently it has a option of closing & minimizing to taskbar notification area.
> 
> I am attaching a screenshot of the UI that pops up when I hover my mouse over one of my fave programs (Auslogics BoostSpeed). Its not just a simple text menu, but an entire UI consisting of graphics, graphs & buttons - all on the notification area.
> 
> ![Screenshot (369)](https://user-images.githubusercontent.com/33699454/131214144-d8bdec60-0ee9-468c-a399-03ce908e1745.png)

@naks110 can you create a discussion in the [Ideas](https://github.com/Splode/pomotroid/discussions/categories/ideas) category please?
